client/src/context/useSocketContext.tsx

```diff
@@ -16,11 +16,16 @@ export const SocketProvider: FunctionComponent = ({ children }): JSX.Element =>
 
   const initSocket = useCallback(() => {
     console.log('trying to connect');
-    setSocket(
-      io('/', {
-        withCredentials: true,
-      }),
-    );
+    const newSocket = io('/', {
+      withCredentials: true,
+    });
+    setSocket(newSocket);
+    newSocket.on('connect', () => {
+      console.log('Socket id is ' + newSocket.id);
```

Comment:
I don't think we need to log to the console when a user connects or disconnect from the socket server 😄

Reply:
Removed console.log